ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -390,24 +402,47 @@ export const Homepage = withUserProfile()(class extends React.Component<
                               <RecentWorkspaces />
                             </FlexColumn>
                             <FlexColumn>
-                              <SmallHeader>Recently Accessed Items</SmallHeader>
-                              <RecentResources/>
+                              {this.state.userHasWorkspaces ? (<React.Fragment>
+                                <SmallHeader>Recently Accessed Items</SmallHeader>
+                                <RecentResources/>
+                              </React.Fragment>) : <div style={{backgroundColor: colors.lightBackground,
```

Comment:
nit: for readability, could we format the ternary a bit differently? Something like:

```
{this.state.userHasWorkspaces ?

(<React.Fragment><SmallHeader>Recently Accessed Items</SmallHeader>
     <RecentResources/></React.Fragment>) : 

<div style={{backgroundColor: colors.lightBackground ..........
```

formatting multi-line ternary https://eslint.org/docs/rules/multiline-ternary